plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/available-filters/checkout-and-place-order-button.md

```diff
@@ -191,16 +191,16 @@ registerCheckoutFilters( 'example-extension', {
 
 ### Description <!-- omit in toc -->
 
-The `placeOrderButtonLabel` filter allows change the label of the "Place order" button.
+The `placeOrderButtonLabel` filter allows change the label of the "Place order · <price/>" button.
```

Comment:
```suggestion
The `placeOrderButtonLabel` filter allows change the label of the "Place order · &lt;price/>" button.
```

The opening bracket should be escaped to ensure that GitHub displays this element. Otherwise, it will be hidden, see:

<img width="701" alt="Screenshot 2024-05-16 at 12 10 35" src="https://github.com/woocommerce/woocommerce/assets/3323310/9c54fdd9-b191-4d14-b5c1-33f514d4ba5e">